.travis.yml

```diff
@@ -4,6 +4,7 @@ language: node_js
 env:
   global:
     - TMPDIR=/tmp
+    - COVERALLS_REPO_TOKEN=vKN3jjhAOwxkv9HG0VBX4EYIlWLPwiJ9d
```

Comment:
Ah I think I need to pass `env` into spawn here: https://github.com/mozilla/fxa-auth-server/pull/1348/files#diff-848030f6d2807bf034c74fb360316d44R15 

updating...